assembly/hdp2.6.3/build/var/lib/ambari-server/resources/common-services/SPLICEMACHINE/2.5.1/service_advisor.py

```diff
@@ -0,0 +1,248 @@
+
+  def getHBaseSiteDesiredValues(self):
+    hbase_site_desired_values = {
+        "hbase.coprocessor.master.classes" : "org.apache.ranger.authorization.hbase.RangerAuthorizationCoprocessor,com.splicemachine.hbase.SpliceMasterObserver",
+        "hbase.regionserver.global.memstore.size" : "0.25",
+        "hfile.block.cache.size" : "0.25",
+        "hbase.regionserver.handler.count" : "200",
+        "hbase.client.scanner.caching" : "1000",
+        "hbase.hstore.blockingStoreFiles" : "20",
+        "hbase.hstore.compactionThreshold" : "5",
+        "hbase.balancer.period" : "60000",
+        "hbase.client.ipc.pool.size" : "10",
+        "hbase.client.max.perregion.tasks" : "100",
+        "hbase.coprocessor.regionserver.classes" : "org.apache.ranger.authorization.hbase.RangerAuthorizationCoprocessor,com.splicemachine.hbase.RegionServerLifecycleObserver",
+        "hbase.hstore.compaction.max.size" : "260046848",
+        "hbase.hstore.compaction.min.size" : "16777216",
+        "hbase.hstore.compaction.min" : "5",
+        "hbase.hstore.defaultengine.compactionpolicy" : "com.splicemachine.compactions.SpliceDefaultCompactionPolicy",
+        "hbase.hstore.defaultengine.compactor" : "com.splicemachine.compactions.SpliceDefaultCompactor",
+        "hbase.coprocessor.region.classes" : "org.apache.hadoop.hbase.security.access.SecureBulkLoadEndpoint,org.apache.ranger.authorization.hbase.RangerAuthorizationCoprocessor,com.splicemachine.hbase.MemstoreAwareObserver,com.splicemachine.derby.hbase.SpliceIndexObserver,com.splicemachine.derby.hbase.SpliceIndexEndpoint,com.splicemachine.hbase.RegionSizeEndpoint,com.splicemachine.si.data.hbase.coprocessor.TxnLifecycleEndpoint,com.splicemachine.si.data.hbase.coprocessor.SIObserver,com.splicemachine.hbase.BackupEndpointObserver",
+        "hbase.htable.threads.max" : "96",
+        "hbase.ipc.warn.response.size" : "-1",
+        "hbase.ipc.warn.response.time" : "-1",
+        "hbase.master.loadbalance.bytable" : "true",
+        "hbase.htable.threads.max" : "96",
+        "hbase.regions.slop" : "0.01",
+        "hbase.regionserver.global.memstore.size.lower.limit" : "0.9",
+        "hbase.client.scanner.timeout.period" : "1200000",
+        "hbase.regionserver.maxlogs" : "48",
+        "hbase.regionserver.thread.compaction.large" : "1",
+        "hbase.regionserver.thread.compaction.small" : "4",
+        "hbase.regionserver.wal.enablecompression" : "true",
+        "hbase.rowlock.wait.duration" : "0",
+        "hbase.splitlog.manager.timeout" : "3000",
+        "hbase.status.multicast.port" : "16100",
+        "hbase.wal.disruptor.batch" : "true",
+        "hbase.wal.provider" : "multiwal",
+        "hbase.wal.regiongrouping.numgroups" : "16",
+        "hbase.zookeeper.property.tickTime" : "6000",
+        "hfile.block.bloom.cacheonwrite" : "TRUE",
+        "io.storefile.bloom.error.rate" : "0.005",
+        "splice.authentication.native.algorithm" : "SHA-512",
+        "splice.authentication" : "NATIVE",
```

Comment:
Does the client get a warning if he changes any of these properties?

Reply:
yes he does if it is different than ours.